rocketc/api_rocket_chat.py

```diff
@@ -256,3 +256,14 @@ def set_custom_fields(self, room_id, custom_fields):
         response = self._request_rocket_chat(method, url_path, data)
         LOG.info("Method set_custom_fields: %s with this data: %s", response, data)
         return response
+
+    def logout_user(self, user_id, login_token):
+        """
+        This method allows to logout an user
+        """
+        url_path = "logout"
+        headers = {"X-Auth-Token": login_token, "X-User-Id": user_id}
+        url = "/".join([self.server_url, self.API_PATH, url_path])
```

Comment:
I think it would be better to use `format` instead of `join`, to construct the API url

Reply:
@diegomillan  why ? in accordance with this page https://gist.github.com/kevinburke/6144826  the join performance is better than format performance, so, is there another reason to prefer format ?

Reply:
In terms of URL construction,  and also for any other person reading the code, format is a cleaner way to do it. On the other hand I don't think that using join will increase the performance in a huge way, since the operation is just a short string construction.